Styles/style.css

```diff
@@ -0,0 +1,165 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500&display=swap');
+
+.container {
+    margin: 50px auto 0;
+}
+
+.main {
+    height: 500px;
+    width: 500px;
+    background-color: #fd5e32;
+    border-radius: 20px;
+    font-family: 'Montserrat', sans-serif;
+    color: #fff;
+}
+
+.leftSide {
+    float: left;
+    text-align: center;
+    width: 225px;
+    height: 475px;
+    background-color: #ff7f50;
+    border-radius: 15px;
+    margin: 12.5px;
+}
+
+.leftSide img {
+    margin: 50px 0;
+    width: 100px;
+    height: 100px;
+    border-radius: 50%;
+    transition-duration: 0.7s;
+}
+
+.leftSide img:hover {
+    transform: rotate(360deg);
+}
+
+.leftSide h1 {
+    font-size: 18px;
+    font-weight: 500;
+    margin-bottom: 20px;
+}
+
+.leftSide h2 {
+    font-size: 16px;
+    font-weight: 400;
+    margin-bottom: 40px;
+}
+
+.buttons {
+    text-align: center;
+}
+
+.buttonLike {
+    font-size: 16px;
+    font-weight: 400;
+    border: 2px solid #fd5e32;
+    color: #fd5e32;
+    border-radius: 20px;
+    background-color: #fff;
+    padding: 10px 20px;
+    line-height: 50px;
+    transition-duration: 1.1s;
+}
+
+.buttonFollow {
+    font-size: 16px;
+    font-weight: 400;
+    border: 2px solid #fd5e32;
+    color: #fd5e32;
+    border-radius: 20px;
+    background-color: #fff;
+    padding: 10px 20px;
+    line-height: 50px;
+    transition-duration: 1.1s;
+}
+
+.buttonMessage {
+    font-size: 16px;
+    font-weight: 400;
+    border: 2px solid #fd5e32;
+    color: #fd5e32;
+    border-radius: 20px;
+    background-color: #fff;
+    padding: 10px 20px;
+    line-height: 50px;
+    margin-bottom: 50px;
+    transition-duration: 1.1s;
+}
+
+.buttonMessage:hover,
+.buttonLike:hover,
+.buttonFollow:hover{
+    transform: rotate(60deg);
+    border: 5px solid #fd5e32;
+    cursor: pointer;
+}
+
+.buttonMessage:hover .span-text,
+.buttonLike:hover .span-text,
+.buttonFollow:hover .span-text{
+    font-weight: 600;
+}
+
+.rightSide {
+    float: right;
+    width: 225px;
+    height: 475px;
+    margin: 12.5px;
+}
+
+.posts {
```

Comment:
also the same styles, to avoid duplicates, you can wrote

.posts,
.likes,
.follower {
  common styles for 3 blocks
}